pom.xml

```diff
@@ -49,6 +49,48 @@
             <version>1.7.30</version>
         </dependency>
 
+        <!-- JUnit5, versions provided using BOM -->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.vintage</groupId>
+            <artifactId>junit-vintage-engine</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
This is to still use maven instead of the new gradle parent plugin?

Reply:
Yes, I wanted to separate concern here.